src/pages/SingleBreedPage/SingleBreed.tsx

```diff
@@ -8,10 +8,43 @@ import BreedData from "../../components/ui/singleBreed/BreedData";
 import Loader from "../../components/ui/loader/Loader";
 import { useAppSelector } from "../../shared/utils";
 
-export default function SingleBreed() {
+export default function SingleBreed({
+  fontSize = "md",
```

Comment:
why do you have a lot of default props here? if it's for storybook to pick it up, it's better to pass these props directly in the story files